resources/views/checkout/steps/success.blade.php

```diff
@@ -1,7 +1,72 @@
 <checkout-success>
     <div dusk="checkout-success">
         <h1 class="font-bold text-4xl mb-5">@lang('Order placed succesfully')</h1>
-        <p>@lang('Partytime!')</p>
-        <p class="animate-spin inline-block text-6xl">🥳</p>
+        <div class="bg-highlight rounded p-8">
+            <p>@lang('We will get to work for you right away')</p>
+            <p>@lang('We will send a confirmation of your order to') <span class="font-bold">@{{$root.user?.email}}</span></p>
+        </div>
+        <div class="mt-4">
+            <div class="flex flex-wrap items-center mb-4 border-b pb-2" v-for="(item, productId, index) in cart.items">
+                <div class="w-1/6 sm:w-1/12 pr-3">
+                    <a :href="item.url" class="block">
+                        <picture>
```

Comment:
Since webp is supported in all major browsers we don't use the picture element anymore.

Reply:
Changed ✅